packages/account-v2/src/containers/Onfido/OnfidoView.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import clsx from 'clsx';
+import { qtMerge } from '@deriv/quill-design';
+import { InlineMessage, useDevice } from '@deriv-com/ui';
+import IcAlertAnnounce from '../../assets/status-message/ic-alert-announce.svg';
+import { OnfidoCustomClass } from './OnfidoView.classnames';
+import './OnfidoView.scss';
+
+type TOnfidoView = {
+    isOnfidoEnabled: boolean;
+    isOnfidoInitialized: boolean;
+    onfidoElementId: string;
+    showStatusMessage: boolean;
+};
+
+export const OnfidoView = ({
+    isOnfidoEnabled,
+    isOnfidoInitialized,
+    onfidoElementId,
+    showStatusMessage,
+}: TOnfidoView) => {
+    const { isMobile } = useDevice();
+    return (
+        <div
+            className={qtMerge('relative min-w-[328px] min-h-[500px] m-auto', OnfidoCustomClass({ mobile: isMobile }))}
+        >
+            <div
+                className={clsx(
+                    'absolute p-400 top-200 min-h-1700 text-center z-[1] w-full [transition:transform_0.35s_linear_4.65s] origin-top',
+                    {
+                        'scale-y-0': showStatusMessage,
+                        invisible: !showStatusMessage,
```

Comment:
Default style (based on default value of showStatusMessage) can go for the base class as they are just boolean. Try using cva if possible

Reply:
Updated.